index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile-Version</title>
+    <link rel="stylesheet" href="main.css">
+  </head>
+  <body>
+    <header class="header">
+      <a href="#"><h4>PEDRO MARQUES</h4></a>
+      <nav><img src="images/menu.svg" alt="Menu"></nav>
+    </header>
+    <section class="grid main">
+      <h1>Hey there. I’m Pedro <span> I’m a software developer</span></h1>
+      <p>
+         I can help you build a product , feature or website Look through
+         some of my work and experience! If you like what you see and 
+         have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <ul>
+        <li><a href=""><img src="images/twitter.svg" alt="social-icons"></a></li>
+        <li><a href=""><img src="images/Linkedin-icon.svg" alt="social-icons"></a></li>
+        <li><a href=""><img src="images/medium.svg" alt="social-icons"></a></li>
+        <li><a href=""><img src="images/github.svg" alt="social-icons"></a></li>
+        <li><a href=""><img src="images/angellist.svg" alt="social-icons"></a></li>
+      </ul>
+        <ul>
+          <li><a href=""><img src="images/Disabled.svg" alt="disabled"></a></li>
+        </ul>
+          <h1>My Recent <br> Works</h1>
+        <ul>
+          <li><a href=""><img src="images/Indicator.svg" alt="indicator"></a></li>
+        </ul>
+    </section>
+    <section class="grid works">
+      <h1>Multi-Post Stories <br> Gain+Glory</h1>
+      <ul>
+        <li>Ruby on rails</li>
+        <li>css</li>
+        <li>JavaScript</li>
+        <li>html</li>
+      </ul>
+      <p>See Project<p>
```

Comment:
- [ ] I believe this section still lacks in formatting. I suggest placing this code section into a project container as illustrated below

```
<div class="project">
    <h2>Multi-Post Stories <br> Gain+Glory</h2>
      <ul>
        <li>Ruby on rails</li>
        <li>css</li>
        <li>JavaScript</li>
        <li>html</li>
      </ul>
      <button>See Project</button>
</div>
```

- [ ] Kindly transform `<h1>Multi-Post Stories <br> Gain+Glory</h1>` into an `<h2>` because it is semantically wrong to have more than one `<h1>` on the same page
- [ ] Kindly make See this project a link a or button tag, and follow the button styling: Active, Hover and Pressed (as seen in the design)
- [ ] Please compare your design and the original figma design and make sure that they are similar in structure and appearance

| Your Design | Figma Design |
| --- | --- |
![image](https://user-images.githubusercontent.com/4247740/159134280-2fdc9962-c4a3-46a1-8aac-3da043c462ba.png) | ![image](https://user-images.githubusercontent.com/4247740/159134300-3ddaf3e7-596e-4351-965e-cc09fd7ae6f9.png) |

- [ ] Also kindly add more projects to this section, It is just one currently